framework/src/iotbus/iotbus_uart.c

```diff
@@ -388,10 +390,24 @@ int iotbus_uart_write(iotbus_uart_context_h hnd, const char *buf, unsigned int l
 	}
 
 	fd = handle->fd;
+
+	tio.loopback = true;
+	ret = loopback(fd, TIOCLOOPBACK, &tio);
```

Comment:
I don't know why user should give this value. When user calls the loopback function, this is fixed and must be given. Then we can do like below
```c
  ret = loopback(fd, &tio);

int loopback(int fd, struct termios *termiosp)
{
	return ioctl(fd, TIOCLOOPBACK, (unsigned long)termiosp);
}
```

Reply:
And same as tio argument.
It only uses the loopback member. Then we can just give int value, not structure.
```c
in os/include/termios.h

/* Constants for use with loopback() */

#define TCLBON    0				/* Enable loopback */
#define TCLBOFF     1				/* Disable loopback */

in loopback.c

int loopback(int fd, int cmd)
{
	return ioctl(fd, TIOCLOOPBACK, (unsigned long)&cmd);
}
```

Reply:
done